project/plugins.sbt

```diff
@@ -1,10 +1,11 @@
 addSbtPlugin("com.github.sbt" % "sbt-unidoc" % "0.5.0")
-addSbtPlugin("org.typelevel" % "sbt-typelevel" % "0.7.7")
-addSbtPlugin("com.github.sbt" % "sbt-ghpages" % "0.8.0")
+addSbtPlugin("org.typelevel" % "sbt-typelevel" % "0.4.22")
```

Comment:
We don't want to downgrade these dependencies. I think they were added as updates originally, but since I merged several PRs to fix the build, these versions are no longer up to date. You should probably be able to leave this file alone, except to add
```scala
addSbtPlugin("org.typelevel" % "sbt-typelevel-mergify" % "0.7.7")
```